packages/flutter/test/material/search_test.dart

```diff
@@ -1232,6 +1261,9 @@ class _TestSearchDelegate extends SearchDelegate<String> {
 }
 
 class _TestEmptySearchDelegate extends SearchDelegate<String> {
+  @override
+  bool? get automaticallyImplyLeading => false;
```

Comment:
This should probably be made variable so all users of `_TestEmptySearchDelegate` do not inherit this value.

Reply:
Yep, I've replaced inheritance with this logic. Now, I just pass this value before using the test. thank you